### PR TITLE
Xantam can no longer escape bans

### PR DESCRIPTION
**Pull request by redmoogle**
### Intent of your Pull Request

Port of https://github.com/yogstation13/yogstation-2017-/pull/3118
https://forums.yogstation.net/index.php?threads/port-this-pr-from-prebase.22010/

There is no escape.

#### Changelog

:cl:  
bugfix: Xantam can no longer escape bans
/:cl:

### CODE REVIEW

File: code/modules/client/client_procs.dm [hunk 1]
```diff
@@ -319,6 +319,13 @@ GLOBAL_LIST_EMPTY(external_rsc_urls)
 			GLOB.mentors += src
 	// yogs end
 
+	if(ckey == "xantam")//"xantam"
+		if(IsBanned("xantam", computer_id, address))
+			webhook_send_asay(ckey, "@Xantam#5689 was just kicked for trying to circumvent a ban on their own server :^)")
+			to_chat(src,"You have been banned! You will be kicked in 5 seconds.") //Kick.
+			message_admins("A banned host tried to connect. They will be kicked in 5 seconds.")
+			to_chat(world, "<span class='warning'>A host was just kicked for trying to circumvent their own server ban! FORSHAME!.</span>")
```
Comment (ynot01) [inline code/modules/client/client_procs.dm]:
```suggestion
			to_chat(world, "<span class='warning'>A host was just kicked for trying to circumvent their own server ban! FORSHAME!</span>")
```

**Comment by redmoogle** [comment]
I literately compiled this goddammit turdis

**Comment by Xantam-13** [comment]
Not sure why we'd get into the habit of trying to attack or troll specific people for no reason over something that isn't an issue. 

Closed.

**Comment by Mqiib** [comment]
Absolute state

**Pull request closed without merging**